core/field_input.ts

```diff
@@ -358,7 +358,7 @@ export abstract class FieldInput<T extends InputTypes> extends Field<
       throw new UnattachedFieldError();
     }
     WidgetDiv.show(this, block.RTL, this.widgetDispose_.bind(this));
-    this.htmlInput_ = this.widgetCreate_() as HTMLInputElement;
```

Comment:
This cast is actually still necessary =) So if you could put it back that'd be great!

Reply:
Okay, will just do that!